numpy/core/fromnumeric.py

```diff
@@ -49,12 +49,14 @@ def _wrapit(obj, method, *args, **kwds):
 
 
 def _wrapfunc(obj, method, *args, **kwds):
+    passkwargs = {k: v for k, v in kwds.items()
+                    if v is not np._NoValue}
```

Comment:
Hmmm, if this is really not here yet, maybe we should just do it in the argmin/argmax function instead.

Reply:
I guess it likely does not matter, just wondering if it adds a largish overhead or not.